edit/buildozer.go

```diff
@@ -355,6 +355,26 @@ func cmdReplace(env CmdEnvironment) (*build.File, error) {
 	return env.File, nil
 }
 
+func cmdSubstitute(env CmdEnvironment) (*build.File, error) {
+	oldRegexp, err := regexp.Compile(env.Args[1])
+	if err != nil {
+		return nil, err
+	}
+	newTemplate := env.Args[2]
+	for _, key := range attrKeysForPattern(env.Rule, env.Args[0]) {
+		attr := env.Rule.Attr(key)
+		if e, ok := attr.(*build.StringExpr); ok {
```

Comment:
optional:  this one is a bit tricky with nested reuse of ok.  I would probably write the same logic as:

e, ok := attr.(*build.StringExpr)
if !ok {
  ListSubstitute(attr, oldRegexp, newTemplate)
  continue
}
if newValue, ok := stringSubstitute(e.Value, oldRegexp, newTemplate); ok {
  env.Rule.SetAttr(key, getAttrValueExpr(key, []string{newValue}))
}

Reply:
Done